### PR TITLE
Cockpit 280

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-280.html

### CODE REVIEW

**Review by mvollmer — APPROVED** [review]

**Comment by mvollmer** [comment]
Looks good to me.

File: _posts/2022-11-16-cockpit-280.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## Try it out
+
+Cockpit 280 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/280)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
```
Comment (garrett) [inline _posts/2022-11-16-cockpit-280.md:28]:
```suggestion
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2022-fbe5e7a3cf)
```

File: _posts/2022-11-16-cockpit-280.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
```
Comment (garrett) [inline _posts/2022-11-16-cockpit-280.md:29]: Still need this URL.

File: _posts/2022-11-16-cockpit-280.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: Cockpit 280
+author: garrett
+date: '2022-11-16'
+tags: cockpit
+slug: cockpit-280
+category: release
+summary: ''
```
Comment (garrett) [inline _posts/2022-11-16-cockpit-280.md:8]:
```suggestion
summary: 'Disallow root login by default'
```

File: _posts/2022-11-16-cockpit-280.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2022-11-16-cockpit-280.md]: I have added it.

**Pull request merged**